hack/generator/Makefile

```diff
@@ -68,4 +72,4 @@ gox: install-tools
 	$(Q) tar -czvf ./bin/$(SHORT_VERSION)/$(APP)_windows_amd64.tar.gz -C ./bin/$(SHORT_VERSION)/ $(APP)_windows_amd64.exe
 
 .PHONY: ci
-ci: all
\ No newline at end of file
+ci: build test-cover
```

Comment:
Best to have the last line ending with LF/CRLF

Reply:
I merged it too quickly sorry. I'll fix these up.

Thank you for the review.